src/test/helpers/navigation.tsx

```diff
@@ -0,0 +1,19 @@
+let params: any = {};
```

Comment:
Can this be a `const`?

Reply:
Cannot, cause the function to reset it needs to change it back to {}